src/main/python/cfn_sphere/stack_configuration/__init__.py

```diff
@@ -129,7 +130,11 @@ def _parse_cli_parameters(parameters):
             try:
                 for key_value_parameter_pair in parameters:
                     stack_and_parameter_key, parameter_value = key_value_parameter_pair.split("=", 1)
-                    stack, parameter_key = stack_and_parameter_key.split(".", 1)
+                    if '.' in stack_and_parameter_key:
+                        stack, parameter_key = stack_and_parameter_key.split(".", 1)
+                    else:
+                        stack = PARAMETERS_FOR_ALL_STACKS
```

Comment:
I really don't like your approach here to set "PARAMETERS_FOR_ALL_STACKS" as stack and thus use the stacks var as transport for an information it is not intended for. I would prefer a dict structure with a global section and a stack specific one.